app/user.go

```diff
@@ -2120,7 +2012,9 @@ func (a *App) PromoteGuestToUser(c *request.Context, user *model.User, requestor
 		mlog.Warn("Failed to get user on promote guest to user", mlog.Err(err))
 	} else {
 		a.sendUpdatedUserEvent(*promotedUser)
-		a.UpdateSessionsIsGuest(promotedUser.Id, promotedUser.IsGuest())
+		if uErr := a.srv.userService.UpdateSessionsIsGuest(promotedUser.Id, promotedUser.IsGuest()); uErr != nil {
+			mlog.Error("Unable to update user sessions", mlog.String("user_id", promotedUser.Id), mlog.Err(uErr))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.

Reply:
Should this be warning?

Reply:
Yep it should 👍